cellprofiler/modules/enhanceedges.py

```diff
@@ -26,6 +27,7 @@
 M_ROBERTS = "Roberts"
 M_LOG = "LoG"
 M_CANNY = "Canny"
+M_KIRSCH = "Kirsch"
```

Comment:
In the future, we should consider using [enumerations](https://www.python.org/dev/peps/pep-0435/) rather than constants (they’ve been [back-ported to 2.7](https://pypi.python.org/pypi/enum34)).